src/test/scala/org/broadinstitute/dsde/workbench/leonardo/auth/SamAuthProviderSpec.scala

```diff
@@ -0,0 +1,14 @@
+package org.broadinstitute.dsde.workbench.leonardo.auth
+
+import org.broadinstitute.dsde.workbench.leonardo.CommonTestData
+import org.scalatest.FlatSpec
+import org.scalatest.concurrent.ScalaFutures
+
+class SamAuthProviderSpec extends FlatSpec with CommonTestData with ScalaFutures {
+  private val samAuth = new SamAuthProvider(config.getConfig("org.broadinstitute.dsde.workbench.leonardo.auth.SamAuthProvider"), serviceAccountProvider)
+
+  "SamAuthProvider" should "construct a sane resourcesApi" in {
```

Comment:
trash this whole file, i just added it in order to be able to breakpoint in tests and step into what swagger-codegen was doing